src/main/java/core/basesyntax/service/impl/FruitTransactionServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionService;
+import core.basesyntax.service.operation.OperationHandler;
+import java.util.List;
+import java.util.Map;
+
+public class FruitTransactionServiceImpl implements FruitTransactionService {
+    private static final String SEPARATOR_IN_LINE = ",";
+    private static final int INDEX_FOR_OPERATION_IN_STRING = 0;
+    private static final int INDEX_FOR_PRODUCT_NAME_IN_STRING = 1;
+    private static final int INDEX_FOR_PRODUCT_VALUE_IN_STRING = 2;
+
+    public FruitTransactionServiceImpl() {
+    }
+
+    @Override
+    public void greatFruitTransaction(List<String> dataFromFile,
```

Comment:
Mehtod greatFruitTransaction parse data and use strategy for operationHandler
You should divide this logic.
```
public interface ParserOperationService {
    List<FruitTransaction> parseContentForOperations(List<String> inputData);
}
```
and
```
public interface FruitShopService {
    void processOfOperations(List<FruitTransaction> fruitTransactions);
}
```